cmd/scepserver/scepserver.go

```diff
@@ -104,19 +104,19 @@ func main() {
 	}
 	allowRenewal, err := strconv.Atoi(*flClAllowRenewal)
 	if err != nil {
-		lginfo.Log("No valid number for allowed renewal time : ", err)
+		lginfo.Log("err", "No valid number for allowed renewal time : " + err.Error())
```

Comment:
can you write these as 
```
lginfo.Log("err", err, "msg", "No valid number for allowed renewal time")
```

Reply:
Done (I'm still learning go-kit, this seems clear to me now though)